### ISSUE TITLE
uograding from 2.4.2 to 2.4.3 breaks build

### ISSUE THREAD

**Issue by jamesarich** [skydoves/landscapist#613]
**Please complete the following information:**
- Library Version [e.g. v1.0.0]
 Version 2.4.3

**Describe the Bug:**
CI fails for this bump. Complaints of missing dependencies on `org.jetbrains.compose` which was not an issue prior (2.4.2)

Add a clear description about the problem.
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processBetaDebugMainManifest'.
> Could not resolve all files for configuration ':app:betaDebugRuntimeClasspath'.
   > Could not find org.jetbrains.compose.runtime:runtime:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
     Required by:
         project :app > com.mikepenz:aboutlibraries-compose-m3:11.2.3 > com.mikepenz:aboutlibraries-compose-m3-android:11.2.3
   > Could not find org.jetbrains.compose.ui:ui:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
     Required by:
         project :app > com.mikepenz:aboutlibraries-compose-m3:11.2.3 > com.mikepenz:aboutlibraries-compose-m3-android:11.2.3
   > Could not find org.jetbrains.compose.foundation:foundation:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
     Required by:
         project :app > com.mikepenz:aboutlibraries-compose-m3:11.2.3 > com.mikepenz:aboutlibraries-compose-m3-android:11.2.3
   > Could not find org.jetbrains.compose.ui:ui:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/ui/ui/1.8.0+check/ui-1.8.0+check.pom
     Required by:
         project :app > com.github.skydoves:landscapist-placeholder:2.4.3 > com.github.skydoves:landscapist-placeholder-android:2.4.3
         project :app > com.github.skydoves:landscapist-glide:2.4.3 > com.github.skydoves:landscapist:2.4.3 > com.github.skydoves:landscapist-android:2.4.3
   > Could not find org.jetbrains.compose.runtime:runtime:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/runtime/runtime/1.8.0+check/runtime-1.8.0+check.pom
     Required by:
         project :app > com.github.skydoves:landscapist-placeholder:2.4.3 > com.github.skydoves:landscapist-placeholder-android:2.4.3
         project :app > com.github.skydoves:landscapist-glide:2.4.3 > com.github.skydoves:landscapist:2.4.3 > com.github.skydoves:landscapist-android:2.4.3
   > Could not find org.jetbrains.compose.foundation:foundation:1.8.0+check.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - file:/home/circleci/src/app/validic-repository/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://jitpack.io/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
       - https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven/org/jetbrains/compose/foundation/foundation/1.8.0+check/foundation-1.8.0+check.pom
     Required by:
         project :app > com.github.skydoves:landscapist-placeholder:2.4.3 > com.github.skydoves:landscapist-placeholder-android:2.4.3
         project :app > com.github.skydoves:landscapist-glide:2.4.3 > com.github.skydoves:landscapist:2.4.3 > com.github.skydoves:landscapist-android:2.4.3
``` 
**Expected Behavior:**
Non breaking update from 2.4.2 -> 2.4.3, or upgrade instructions.
A clear description of what you expected to happen.

**Comment by skydoves** [comment]
Hey @jamesarich, thank you for your reporting! Just released a new version 2.4.4 and it should work as expected.

**Comment by jamesarich** [comment]
Thanks for the quick work @skydoves- 2.4.4 builds fine 🎉
